cromshell

```diff
@@ -621,7 +658,17 @@ function submit()
 
   assertCanCommunicateWithServer $CROMWELL_URL
 
-  local response=$(curl --connect-timeout $CURL_CONNECT_TIMEOUT --max-time $CURL_MAX_TIMEOUT -s -F workflowSource=@${1}  ${2:+ -F workflowInputs=@${2}} ${3:+ -F workflowOptions=@${3}} ${4:+ -F workflowDependencies=@${4}} ${CROMWELL_URL}/api/workflows/v1)
+  local wdl=${1}
+  local json=${2}
+  local optionsJson=${3}
+  local dependenciesZip=${4}
+
+  assertRequiredFileIsNonEmpty "$wdl" "wdl"
```

Comment:
For style I'd address all these variables with braces ala `${VARIABLE}`.  

I have a style / whitespace branch open but I'd like to start enforcing conventions.